pkg/kapp/cmd/appgroup/deploy.go

```diff
@@ -68,11 +68,16 @@ func (o *DeployOptions) Run() error {
 		return err
 	}
 
+	var exitCode float64 = 0
 	// TODO is there some order between apps?
 	for _, appGroupApp := range updatedApps {
 		err := o.deployApp(appGroupApp)
 		if err != nil {
-			return err
+            if deployErr, ok := err.(cmdapp.DeployDiffExitStatus); ok {
```

Comment:
nit: looks like formatting is a bit off

Reply:
![image](https://user-images.githubusercontent.com/1926750/124498179-37853900-ddbc-11eb-939d-679d68d7e773.png)
I think the diff view is misleading, also the linters don't indicate anything malformed.

Reply:
it seems that if is indented via spaces whereas the rest of the code is indented via tabs, and thats why it shows up at different invent levels in github and not in your editor.

Reply:
replaced whitestaces by tabs.

does anything speak against adding - gofmt to the linters section as part of the .golangci.yml?